scripts/prepare

```diff
@@ -4,8 +4,8 @@ set -e
 # Pull in the upstream charts
 for f in packages/*; do
 	if [[ -z $CHART || $CHART == $(basename -- ${f}) ]]; then
-		if [[ -f ${f}/package.yaml ]]; then
-			url=$(cat ${f}/package.yaml | yq r - url)
+		url=$(cat ${f}/package.yaml | yq r - url)
+		if [[ ( -f ${f}/package.yaml ) && ! ( -z ${url} ) ]]; then
```

Comment:
continue would skip the rest of the steps in the steps, so prepare-crds would never get called. I opted to use some of your suggestion to still set url once, and then check if its null as part of the if statement.

Reply:
Sounds good to me. Resolving both this and https://github.com/rancher/charts/pull/607#discussion_r483292418 since they are linked